postgres/content.md

```diff
@@ -180,7 +180,15 @@ ENV LANG de_DE.utf8
 
 Since database initialization only happens on container startup, this allows us to set the language before it is created.
 
-Also of note, Alpine-based variants do *not* support locales; see ["Character sets and locale" in the musl documentation](https://wiki.musl-libc.org/functional-differences-from-glibc.html#Character-sets-and-locale) for more details.
+Also of note, Alpine-based variants starting with Postgres 15 support [ICU locales](https://www.postgresql.org/docs/15/locale.html#id-1.6.11.3.7). Previous Postgres versions based on alpine do *not* support locales; see ["Character sets and locale" in the musl documentation](https://wiki.musl-libc.org/functional-differences-from-glibc.html#Character-sets-and-locale) for more details.
+
+You can extend the Alpine-based images with a simple `Dockerfile` to set a different locale. The following example will set the default locale to `de_DE.utf8`:
+
+```dockerfile
+FROM %%IMAGE%%:15-alpine
+ENV POSTGRES_INITDB_ARGS "--locale-provider=icu --icu-locale=de-DE"
+ENV LANG de_DE.utf8
+```
```

Comment:
````suggestion
```console
$ docker run -d -e LANG=de_DE.utf8 -e POSTGRES_INITDB_ARGS="--locale-provider=icu --icu-locale=de-DE" -e POSTGRES_PASSWORD=mysecretpassword %%IMAGE%%:15-alpine 
```
````
Could just be a `docker run` instead? I'm not certain it needs to be one or the other. I default to a `docker run` for things that can reasonably be done with it (i.e. when not installing packages or modifying files).